compositor/meson.build

```
@@ -0,0 +1,57 @@
+cc = meson.get_compiler('c')
+
+wayland_server = dependency('wayland-server')
+wayland_client = dependency('wayland-client')
+wayland_cursor = dependency('wayland-cursor')
+wayland_egl    = dependency('wayland-egl')
+wayland_protos = dependency('wayland-protocols', version: '>=1.14')
+xkbcommon      = dependency('xkbcommon')
+xcb            = dependency('xcb', required: true)
```

```suggestion
xcb            = dependency('xcb')
```
required is the default, right?

---

New to meson, yes it is :)